features/checkout/addressing_order/filling_shipping_and_billing_address_details.feature

```diff
@@ -7,14 +7,16 @@ Feature: Addressing an order
     Background:
         Given the store operates on a single channel in "France"
         And the store has a product "PHP T-Shirt" priced at "$19.99"
+        And the store has "Raven Post" shipping method with "€10.00" fee
```

Comment:
OK, I see, but maybe we can use ``And the store ships everything for free`` as the fee is not relevant to this scenario.